charts/tyk-operator/templates/deployment.yaml

```diff
@@ -0,0 +1,105 @@
+---
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
I wonder is this a CRD? Could you share what's the source of this chart?

I did this and seems it got a diff chart:
```
helm repo add tyk-helm https://helm.tyk.io/public/helm/charts/
helm repo update
helm pull tyk-helm/tyk-oss 
```

the above shows a ~5k lines CRD's file 🤨